libr/anal/sign.c

```diff
@@ -1480,6 +1469,35 @@ struct ctxListCB {
 	PJ *pj;
 };
 
+static void list_sanitise_warn(char *s, const char *name, const char *field) {
```

Comment:
see r_name_filter() and such

Reply:
I will update to use `r_name_filter` on `it->name` and `it->space->name`.

I will use this `list_sanitize_warn` for sig types, like bytes, bbhash and vars. This is because `r_name_filter` does not allow names to start with a number.

In the future I can add a `RAsignItem` validator or sanitizer so it will all be done in one place. Does that sound good?